src/disconnect.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo -e 'disconnect 5C:BA:37:26:6D:9A\nquit' | bluetoothctl
+
```

Comment:
Same issues here as the previous script. It would be nice if these were both in the same scripts and we could just decide what function to use by passing arguments.